.github/workflows/unit_tests.yml

```
@@ -30,21 +30,20 @@ jobs:
         with:
           python-version: ${{ matrix.python }}
           cache: 'pip'
-          cache-dependency-path: '**/setup.cfg'
+          cache-dependency-path: '**/pyproject.toml'
       - name: Install package with dependencies
         run: |
-          pip install -e ".[dev]"
-          python -m pip install tox tox-gh-actions
+          pip install -e ".[test]"
         if: steps.python-cache.outputs.cache-hit != 'true'
```

actionlint

property "python-cache" is not defined in object type {}

(expression)